src/styles/createPalette.js

```diff
@@ -13,6 +16,42 @@ import {
     teal,
 } from '../colors';
 
+export const light = {
+    background: {
+        default: common.white,
+        contrastPrimary: grey[600],
+        light: grey[100],
+        paper: common.white,
+        primary: common.white,
+        secondary: grey[200],
+    },
+    divider: grey[300],
+    text: {
+        active: cyan[500],
+        contrastText: common.white,
+        disable: grey[300],
+        inverseDisable: grey[500],
+        link: cyan[500],
+        primary: grey[600],
+        secondary: grey[400],
+    },
+    action: {
+        disabled: grey[300],
+    },
+};
+
+/**
+ * TODO: One day being able to switch to a dark theme for user's eyes and batteries would be
+ * an amazing nice to have. For now `dark` is the exact same palette as light.
+ */
+export const dark = {
+    text: {
+        disable: light.text.disable,
+        primary: common.white,
+        secondary: light.text.secondary,
+    },
+};
```

Comment:
If an when we do make dark theme available, will it need more settings, similar to what light theme has above?

Reply:
Correct, it will. For now it doesn't do much what-so-ever. I needed the dark const defined so that some other funcs that Material UI need so the code would not fail. Kind of a bummer I had to do this, but at least it sets us up for some success down the road.

Reply:
Sounds good.  Funnily enough, @Benjamin-Gnewuch was asking on a call yesterday if we ever planned on doing a dark theme for HC, and I told him that, serendipitously I had reviewed this PR and seen this. :-)

Reply:
A dark theme would be great to have. Easier on the eyes and people's laptop batteries.